src/components/base/FormattedVal/index.js

```diff
@@ -131,6 +140,11 @@ function FormattedVal(props: Props) {
       )}
     </T>
   )
+
+  if (showTooltip) {
+    return <Tooltip render={() => unit && formatCurrencyUnit(unit, val)}> {content} </Tooltip>
```

Comment:
should the `unit` check be done in the if? because as far as i read it, if there is no unit, we would have no tooltip right?

```
if (showTooltip && unit) {
    return <Tooltip render={() => formatCurrencyUnit(unit, val)}> {content} </Tooltip>
}
```

I imagine this case never present, but code is easier to understand